doc/source/rst.rst

```diff
@@ -2,14 +2,174 @@
 Understanding the RST
```

Comment:
please add also info about how to work with intersection links within same page or intesection links from other page but in same document.
How to use anchors , etc
Also how to work with ordered and unordered list.